recipes/default.rb

```diff
@@ -44,7 +44,7 @@
 
     require 'digest/sha1'
 
-    node['papertrail']['watch_files'].each do |filename, tag|
+    DirHelper.expand(node['papertrail']['watch_files']).each do |filename, tag|
```

Comment:
i've changed the behavior so globbing just happens when there's something to glob, so we can have the goods from both worlds ;)